### PR TITLE
PAY-687 Add new UI to tipping linking to the BuyAudio flow

### PR DESCRIPTION
**Pull request by rickyrombo**
### Description

[Figma](https://www.figma.com/file/kDAWdnLlorKQtAF4pdc0cB/Buy-%24AUDIO-CTA-in-the-Tipping-Flow?node-id=2%3A2171)

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

*Please describe the tests that you ran to verify your changes. Provide repro instructions & any configuration.*

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/mjp-stripe-tips

File: packages/web/src/components/tipping/tip-audio/SendTip.tsx [hunk 1]
```diff
@@ -163,14 +171,43 @@ export const SendTip = () => {
   )
 
   return receiver ? (
-    <div className={styles.container}>
+    <div
```
Comment (piazzatron) [inline packages/web/src/components/tipping/tip-audio/SendTip.tsx:174]: nit: maybe worth pulling this out into a render fn with a few separate returns, this nested ternary is getting a bit hairy

**Review by piazzatron — COMMENTED** [review]
I think we need to feature flag this 👀

**Comment by piazzatron** [comment]
Let's also figure out how to deal with these triple nested modals! :D

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/mjp-stripe-tips

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/mjp-stripe-tips

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/mjp-stripe-tips

**Review by sddioulde — APPROVED** [review]
Looks lit

**Pull request merged**